test/yamlLanguageService.test.ts

```diff
@@ -39,7 +39,7 @@ describe('getLanguageService()', () => {
       // Setup object that resolves schema content
       schemaContentMap[schemaUri] = `
         {
-            "$schema": "https://json-schema.org/draft/2020-12/schema",
+            "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
This is the part that scares me about the change, this now errors out because the $schema field actually gets checked by tha `validateSchema` method.

Reply:
I went ahead and added support for the rest of the drafts from this and left the test as it is.